### PR TITLE
CreditCardMask

### PR DESCRIPTION
**Pull request by shishkina**
I only added one folder. I added solution just in case as well. Not sure if that is the proper way.

### CODE REVIEW

File: challenges/mode/modequestion.js [hunk 1]
```diff
@@ -6,6 +6,18 @@
 function SimpleMode(arr) {
 
   // code goes here
+  var mode = 0;
```
Comment (sarahzinger) [inline challenges/mode/modequestion.js]: @shishkina did you intend to add this file?

File: challenges/multiplesOf3And5/multiplesOf3And5.md [hunk 2]
```diff
@@ -3,3 +3,16 @@
 "If we list all the natural numbers below 10 that are multiples of 3 or 5, we get 3, 5, 6 and 9. The sum of these multiples is 23.
 
 Find the sum of all the multiples of 3 or 5 below 1000."
+
```
Comment (sarahzinger) [inline challenges/multiplesOf3And5/multiplesOf3And5.md]: @shishkina again i think you maybe didn't intend to add this file :)

File: challenges/primeFactors/primeFactors.md [hunk 3]
```diff
@@ -3,3 +3,44 @@
 "The prime factors of 13195 are 5, 7, 13 and 29.
 
 What is the largest prime factor of the number 600851475143 ?"
+
```
Comment (sarahzinger) [inline challenges/primeFactors/primeFactors.md]: @shishkina one more accidental add!

**Comment by sarahzinger** [comment]
@shishkina I love your credit card problem and would love to have it in for next month's event! I think you may have accidentally added some partial solutions that you were working on to some of the old question files. If you can it would be awesome if you could make a pull request (or edit this one!) with just the changes you want to add to the repo. Let me know if you run into problems!

**Comment by shishkina** [comment]
All done! Thank you very much! This pull request is modified.

**Comment by sarahzinger** [comment]
awesome!! 💯

**Pull request merged**